src/utilities/createRouterReject.ts

```diff
@@ -47,26 +61,6 @@ export function createRouterReject({
     return components[type]
   }
 
-  const getRejectionRoute: GetRejectionRoute = (type) => {
-    const route = {
-      name: type,
-      path: '',
-      component: markRaw(getRejectionComponent(type)),
-    }
-
-    const resolved: Resolved<Route> = {
-      matched: route,
-      matches: [route],
-      name: type,
-      depth: 0,
-      path: '',
-      query: '',
-      params: {},
-    }
-
-    return resolved
-  }
-
   const clearRejection: ClearRejection = () => {
```

Comment:
I also liked how this change simplifies this contract, no longer needs to export `getRejectionRoute`, which was unique to NotFound

Reply:
So the problem with this is it doesn't consider that the user can provide their own component. Which is basically the only reason this utility existed in the first place.

Reply:
I don't see how I broke that functionality? Tests all passed?

Reply:
That's because I don't think there's a test for that lol. The rejection works just fine. But if you were to look at router.route for the not found route you'd see the library's component and not the custom component.